lib/query_components/text_query.rb

```diff
@@ -51,6 +63,12 @@ def all_searchable_text_query
       dis_max_query(queries, tie_breaker: 0.1)
     end
 
+    def quoted_phrase_conditions
+      groups = [ field_boosts_phrase ]
+      dismax_groups(groups)
```

Comment:
If there's only a single item in the list, there's no benefit to putting it through dismax_groups.